.github/workflows/tests.yml

```diff
@@ -43,12 +43,12 @@ jobs:
   linkcheck:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
 
-      - name: Set up Python 3.8
-        uses: actions/setup-python@v1
+      - name: Set up Python 3.11
+        uses: actions/setup-python@v4
         with:
-          python-version: 3.8
+          python-version: 3.11
```

Comment:
In https://github.com/minddistrict/sphinxcontrib-httpdomain/blob/dacbd22ea917c6d995e4d518d9f1568c77e0e4ab/tox.ini we do linkcheck only with py37, here with py311. Might this be the reason it is not working? Or is the test failure of `api/github` really a wrong link?

Reply:
So you skipped the link check now?

Reply:
This is really a wrong link. It might have worked once but also does not work locally without the Python version mismatch.